framework/src/main/java/org/checkerframework/framework/flow/CFAbstractStore.java

```diff
@@ -71,6 +73,71 @@ public abstract class CFAbstractStore<V extends CFAbstractValue<V>, S extends CF
   /** Information collected about local variables (including method parameters). */
   protected final Map<LocalVariable, V> localVariableValues;
 
+  /** Stores after boolean variable assignment. */
```

Comment:
I wound find this comment more useful if it said more about the semantics, rather than when the field is first set.  The map is from *boolean* local variables, and the mapped-to store indicates facts that are true depending on whether the boolean local variable is true or false.